modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-api -o yaml
+
+oc -n $NAMESPACE describe deploy virt-api
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+//VirtApiRESTErrorsBurst Alert
+[id="VirtApiRESTErrorsBurst_{context}"]
+== VirtApiRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls fail in virt-api in the last five minutes.
+
+.Reason
+
+A very high rate of failed REST calls to virt-api causes slow response, slow execution of API calls, or even complete dismissal of API calls.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Check to see how many running virt-api pods exist.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get pods -l kubevirt.io=virt-api
+----
+
+. View the pods' logs using `oc logs` and pod status using `oc describe`.
+
+.Resolution
+
+Virt-api pods can be down for several reasons. Identify the root cause and take appropriate action.
+
+. Check the status of the virt-api deployment to find out more information. These commands provide the associated events and show if there are any issues with pulling an image or a crashing pod.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-api -o yaml
+oc -n $NAMESPACE describe deploy virt-api
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+//VirtControllerDown Alert
+[id="VirtControllerDown_{context}"]
+== VirtControllerDown alert
+
+.Description
+
+If no detection of virt-controllers occurs in the past five minutes, then virt-controller deployment has a default replica of two pods.
+
+.Reason
+
+If the virt-controller fails, then VM lifecycle management tasks, such as launching a new VM instance or shutting down an existing VM instance, completely fail.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Check the status of the virt-controller deployment.
++
+[source,terminal]
+----
+oc get deployment -n $NAMESPACE virt-controller -o yaml
+----
+
+. Check the manager pod's logs to determine why creating the virt-controller pods fails.
++
+[source,terminal]
+----
+oc get logs <virt-controller-pod>
+----
+
+[NOTE]
+====
+Do not use a name for the virt-controller pod that is similar to virt-controller-7888c64d66-dzc9p.
+====
+
+There may be several pods that run virt-controller.
+
+.Resolution
+
+There are several reasons why detection of no running virt-controller occurs. Identify the root cause and take appropriate action.
+
```

Comment:
"reasons why" --> "known reasons (listed as below) why the"

Reply:
@yuhaohaoyu Changed to "There are several known reasons why the detection of no running virt-controller occurs. Identify the root cause from the list of possible reasons and take appropriate action."

We try to avoid directional words like "above" or "below" in downstream docs. Does this change work?

Reply:
> @yuhaohaoyu Changed to "There are several known reasons why the detection of no running virt-controller occurs. Identify the root cause from the list of possible reasons and take appropriate action."
> 
> We try to avoid directional words like "above" or "below" in downstream docs. Does this change work?

Understood. @mgarrellRH Thanks for explanation. The change works for my view.